response_operations_ui/forms.py

```diff
@@ -199,6 +199,7 @@ class CreateSurveyDetailsForm(FlaskForm):
                                                 message='Please use alphanumeric characters only.')])
     survey_ref = StringField('survey_ref', validators=[InputRequired(message="Please remove spaces in Survey ID")])
     legal_basis = SelectField('legal_basis', choices=[('', 'Select an option')])
+    survey_mode = SelectField('survey_mode', choices=[('', 'Select an option'), ('eQ', 'eQ'), ('SEFT', 'SEFT')])
```

Comment:
is one of the pairs of values the value of the field?  If so, one of them needs to be EQ.  I think you said you'd look at it, but it's important that it gets done so I'll mention it so it doesn't get forgotten about!